_guidelines-v4/01-introduction/02-basicConcepts/01-musicaldomains.md

```diff
@@ -0,0 +1,27 @@
+---
+version: "v4"
+title: "Musical Domains"
+sectionid: "musicalDomains"
+---
+
+The term "music" has many different notions, ranging from audible sounds over written performance instructions or transcriptions of such events to conceptual rulesets that establish different theories of what music is, and what is allowed in music. In 1965, Milton Babbitt distinguished between _graphemic_, _acoustic_ and _auditory_ aspects of music (Babbitt, Milton: _The Use of Computers in Musicological Research_, in: _Perspectives of New Music_ 3/2 (1965), p. 74–83).
```

Comment:
typographic quotation marks “…”

Babbitt: if you want the precise page: 76

Reply:
done, thanks for the ref